profiler/src/Demos/Samples.ExceptionGenerator/Properties/launchSettings.json

```diff
@@ -1,21 +1,32 @@
 {
   "profiles": {
-    "Demos.ExceptionGenerator": {
+    "ExceptionGenerator-Alone": {
+      "commandName": "Executable",
+      "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\Samples.ExceptionGenerator\\$(TargetFramework)\\Samples.ExceptionGenerator.exe",
+      "nativeDebugging": true
+    },
+    "ExceptionGenerator+Profiler": {
       "commandName": "Executable",
       "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\Samples.ExceptionGenerator\\$(TargetFramework)\\Samples.ExceptionGenerator.exe",
       "environmentVariables": {
-        "COR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "COR_ENABLE_PROFILING": "1",
-        "CORECLR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
+        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
+        "COR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
+        "COR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
+
         "CORECLR_ENABLE_PROFILING": "1",
-        "DD_DOTNET_PROFILER_HOME": "$(BuildOutputRoot)\\bin\\$(Configuration)-AnyCPU\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Managed\\",
-        "COMPlus_EnableDiagnostics": "1",
         "CORECLR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
+        "CORECLR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "CORECLR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
+
+        "COMPlus_EnableDiagnostics": "1",
+
         "DD_PROFILING_ENABLED": "1",
+        "DD_PROFILING_CPU_ENABLED": "1",
+        "DD_PROFILING_EXCEPTION_ENABLED": "1",
```

Comment:
maybe?
```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

Reply:
I added only the interesting profilers (no allocations here)